com.unity.ml-agents/Runtime/Policies/BarracudaPolicy.cs

```diff
@@ -55,10 +60,21 @@ public void RequestDecision(AgentInfo info, List<ISensor> sensors)
         }
 
         /// <inheritdoc />
-        public float[] DecideAction()
+        public ref readonly ActionBuffers DecideAction()
```

Comment:
How is this going to work with multiple actuators and mixed action space types? Will we do all the continuous actions, then all the discrete ones?

Reply:
Never mind, makes more sense after re-reading ExecuteActions